### DISCUSSION TITLE
mindmap for marp

### DISCUSSION THREAD

**Discussion by GXY2017** [marp-team/marp#411] [Q&A] [answered]
hi, I am looking for a mindmap extension for marp in VS code. I tried Markdown Preview Markmap Support, but it does not work.

Any other tool I can use, or which syntax MARP follows?

**Comment by GXY2017** [comment]
I found MarkMap in VS code extension,  it works in a separate file.  How to import it into MARP and apply to one slide?

**Comment by yhatt** [comment, marked as answer]
If you want to stick to VS Code (GUI interface), there is no way to combine MarkMap and Marp into one file. You should export MarkMap to SVG image, and use it in Marp Markdown via Markdown image syntax `![](markmap.svg)`.

If you are open to CLI, you can probably render MarkMap diagrams written in your Marp Markdown directly, by using [a possibly compatible markdown-it plugin](https://www.npmjs.com/search?q=markdown-it%20markmap). See https://github.com/marp-team/marp-cli#functional-engine about how to use Marp/markdown-it plugins with Marp CLI. _(Disclaimer: I have not tested)_
